src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -302,14 +302,49 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ## Query parameters
 
+    When query parameters are passed to an action, you have access to these from the `params` method.
```

Comment:
I'm not sure this belongs here. You can use `params.get` for any kind of param not just query params. Maybe a new section like `Dynamic param access` or something?

Reply:
Yeah, we probably need a whole section to talk about the `from_json`, `from_query`, etc... I'll need to think about what the looks like.

Reply:
Oh yeah I forgot about those 😂 So many new things!

Reply:
Ok, I'm going to refactor this section a little to add some more info.